EZEE-3289 - DFS configuration change (#1157)
ezsystems/developer-documentation: docs/guide/clustering.md

You need to configure both metadata and binarydata handlers.

> EZ Platform ships with a custom local adapter (`ibexa.platform.io.nfs.adapter.site_access_aware`), which
> decorates the Flysystem local adapter to enable support for SiteAccess-aware settings.

Reviewer: ```suggestion
eZ Platform ships with a custom local adapter (`ibexa.platform.io.nfs.adapter.site_access_aware`),
which decorates the Flysystem local adapter to enable support for SiteAccess-aware settings.
```

Reviewer 2: If we're strict on the names, for <=3.2 it should be "eZ Platform by Ibexa" ;)

> The custom local adapter must be used instead of the Flysystem local adapter if an NFS path relies on

Reviewer: ```suggestion
You must use the custom local adapter instead of the Flysystem local adapter if an NFS path relies on
```

    dbal:
        connections:
            dfs:
                # configure these settings to match your database server
                driver: pdo_mysql
                charset: utf8mb4
                default_table_options:
                    charset: utf8mb4
                    collate: utf8mb4_unicode_520_ci
                url: mysql://root:rootpassword@127.0.0.1:3306/ezdfs

# define the Flysystem handler

> oneup_flysystem:

Reviewer 2: This configuration also was changed, 3 lines below - I don't have a way to mark it. See the original PR ezsystems/ezplatform#596. You also need to include `parameters` section at the top, but just `ibexa.platform.io.nfs.adapter.config`, since the other two params are not used for this snippet.
